### PR TITLE
new lightning wrapper params

### PR DESCRIPTION
**Pull request by VictorGillioz**
PR for a new dynamic in the lightning wrapper parameters. We now give the model Class and model parameters to the wrapper.

This has multiple advantages:
- Modularity between wrapper and model is preserved
- Wrapper and model are instantiated together
- The wrapper is able to save the model parameter as hyperparameters
- The instantiated model is not saved to the checkpoints (this usually throws a warning due to bad memory use)
- We can still load the wrapper without providing the hyperparameters

### CODE REVIEW

**Review by AugustoPeres — APPROVED** [review]
LGTM!! Thanks Victor, this should work nicely

File: meshnets/utils/model_training.py [hunk 1]
```diff
@@ -78,28 +78,25 @@ def train_model(config, experiment_config, train_dataset, validation_dataset):
                                    persistent_workers=True,
                                    shuffle=False)
 
-    # Define the model
-    model = MeshGraphNet(
+    # Define the wrapper and the model
+    lightning_wrapper = MGNLightningWrapper(
```
Comment (sarmento) [inline meshnets/utils/model_training.py:82]: try just importing meshnets and expand the the path here to include all the packages.
This allow us to detect redundancies in the name of the pacakages and in the name of the classes.

**Review by sarmento — APPROVED** [review]
This looks good.
As you go along, try NOT to import classes directly.

**Comment by VictorGillioz** [comment]
Thanks @sarmento for the suggestion. I saw @hpenedones ’s mail and I am planning to change all the imports in the project soon. I will do a separate PR for that

**Pull request merged**